src/main/java/graphql/execution/Execution.java

```diff
@@ -39,9 +39,9 @@
     private final Instrumentation instrumentation;
 
     public Execution(ExecutionStrategy queryStrategy, ExecutionStrategy mutationStrategy, ExecutionStrategy subscriptionStrategy, Instrumentation instrumentation) {
-        this.queryStrategy = queryStrategy != null ? queryStrategy : new SimpleExecutionStrategy();
-        this.mutationStrategy = mutationStrategy != null ? mutationStrategy : new SimpleExecutionStrategy();
-        this.subscriptionStrategy = subscriptionStrategy != null ? subscriptionStrategy : new SimpleExecutionStrategy();
+        this.queryStrategy = queryStrategy != null ? queryStrategy : new AsyncExecutionStrategy();
+        this.mutationStrategy = mutationStrategy != null ? mutationStrategy : new AsyncExecutionStrategy();
```

Comment:
yes ... you are right ... too many different places where we use a default strategy.